docs/development-environment.md

```diff
@@ -109,14 +109,15 @@ If you're not familiar with PHP Unit Testing, you can also check [this tutorial]
 
 ## Unit-testing the JS Admin Page
 
-You can run [Mocha](https://mochajs.org/) based tests for the Admin Page source code.
+You can run [Mocha](https://mochajs.org/) based tests for the Admin Page source code and the modules Javascript code.
```

Comment:
This is worth rewording to be more general. Also, it's under the heading `Unit-testing the JS Admin Page`

Reply:
Good point, I'll reword this. Thanks!

Reply:
Done!